.github/workflows/e2e.yml

```diff
@@ -18,13 +18,13 @@ jobs:
         timeout-minutes: 15
         runs-on: ubuntu-latest
         steps:
-            - uses: actions/setup-node@v4
-              with:
-                  node-version: 18
-
             - name: Checkout code
               uses: actions/checkout@v4
 
+            - uses: actions/setup-node@v4
+              with:
+                  node-version-file: '.nvmrc'
```

Comment:
Great idea!

Reply:
Yeah, now we only have one place to set the node version, which is good for future maintenance.